app/backend/src/couchers/servicers/api.py

```diff
@@ -423,12 +423,17 @@ def ListFriends(self, request, context):
         users2 = aliased(User)
 
         with session_scope() as session:
+            user = get_user_by_field(session, str(context.user_id))
```

Comment:
Please don't use this function except in the few places where we take user input (login/user's page).

There's a standard `user = session.query(...).one()` query